docs/UserGuide.md

```diff
@@ -74,6 +74,17 @@ A pop-up of a new window that lists out all the main commands with their respect
 
 ![help window](images/helpWindow.png)
 
+### Viewing calendar: `calendar`
+
+Shows a calendar window showing any client's appointment date.
+
+Precise command format: `calendar`
```

Comment:
paiseh syam, we're sticking to Format, not precise command format. can refer to the userguide in the mater branch for more details! look at the addcommand part